pkgs/applications/version-management/git/default.nix

```diff
@@ -382,8 +379,7 @@ stdenv.mkDerivation (finalAttrs: {
         ''
       else
         ''
-          # replace git-svn by notification script
-          notSupported $out/libexec/git-core/git-svn
+          rm -f $out/libexec/git-core/git-svn
```

Comment:
small note: would we not want to use -f here to know when it fails because the file moved or something else changed?

Reply:
Good question. I used `rm -f` here since it's replacing `unlink <file> || true`, and my goal wasn't to change the fundamental build behaviour, but I'll check now to see if removing the `-f` is viable…

Reply:
Thanks for testing! Let's hope this is now super solid to merge!